.github/workflows/ci.yml

```diff
@@ -1,49 +1,59 @@
 name: CI
 
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
+on: push
```

Comment:
For open-source projects like this, the `push` event isn’t enough. If someone opens a pull request from a fork, the `push` events triggers actions on that fork, but not on the main repo. I usually use these settings:
```yaml
on:
  pull_request:
  push:
    branches:
      - main
```